api/demo/init.go

```diff
@@ -0,0 +1,66 @@
+package demo
+
+import (
+	"github.com/pkg/errors"
+	portainer "github.com/portainer/portainer/api"
+	"github.com/portainer/portainer/api/dataservices"
+)
+
+func initDemoUser(
+	store dataservices.DataStore,
+	cryptoService portainer.CryptoService,
+) error {
+
+	password, err := cryptoService.Hash("tryportainer")
+	if err != nil {
+		return errors.WithMessage(err, "failed creating password hash")
+	}
+
+	admin := &portainer.User{
+		Username: "admin",
+		Password: password,
+		Role:     portainer.AdministratorRole,
+	}
+
+	err = store.User().Create(admin)
+	return errors.WithMessage(err, "failed creating user")
```

Comment:
`store.User().Create(admin)`  set's `admin''s Id that we can return

Reply:
done